src/main/java/hudson/plugins/git/ChangelogToBranchOptions.java

```diff
@@ -34,7 +34,7 @@ public String getCompareTarget() {
         return compareTarget;
     }
 
-    public String getRef() {
+    public String getRevision() {
```

Comment:
This changes the public API.  That seems unnecessary, even if it is believed there are no consumers of the public API.

Is there a compelling reason to change the public API?

Unless there is such a reason, please retain the prior method name.

Reply:
The main reason for changing it is that the current name is a misnomer.